freqtrade/tests/test_freqtradebot.py

```diff
@@ -999,7 +999,8 @@ def test_execute_sell_up(default_conf, ticker, ticker_sell_up, mocker) -> None:
     mocker.patch.multiple(
         'freqtrade.freqtradebot.exchange',
         validate_pairs=MagicMock(),
-        get_ticker=ticker
+        get_ticker=ticker,
+        get_fee=MagicMock(return_value=0.0025)
```

Comment:
Parametrize - this MagicMock is used at least 3 times in this file alone ....

Reply:
Fixed https://github.com/gcarq/freqtrade/pull/603/commits/cc5991d269f260e54c04d80f8f47fcdaa1642e83